apps/ledger-live-mobile/e2e/specs/password.spec.ts

```diff
@@ -8,10 +8,6 @@ import { loadConfig } from "../bridge/server";
 const CORRECT_PASSWORD = "passWORD$123!";
 
 describe("Password Lock Screen", () => {
-  beforeAll(async () => {
-    await device.launchApp();
```

Comment:
I see this is in the `beforeAll` now. How do we make sure the app is in a clean state for each test?

Reply:
https://wix.github.io/Detox/docs/next/api/device-object-api/#1-newinstancelaunching-a-new-instance-of-the-app